### PR TITLE
GH-109975: Copyedit 3.13 What's New: Improved Modules

### PR DESCRIPTION
**Pull request by AA-Turner**
A copy-editing pass for *improved modules*. Next will be *optimised modules*. Sorry for the slight delay, I've been travelling.

Summary of changes (by theme):

* Link module headings to their module
* Reduce duplication by not listing the module name in cross-links
* Provide links or references to concepts where appropriate
* Word items by the intended change first, describing the implementation second, rather than vice-versa.
* Remove double spaces, `()` in cross-reference roles, etc
* General wording improvements

A

* Issue: gh-109975

----
📚 Documentation preview 📚: https://cpython-previews--123132.org.readthedocs.build/en/123132/whatsnew/3.13.html#improved-modules

### CODE REVIEW

File: Doc/whatsnew/3.13.rst [hunk 1]
```diff
@@ -190,7 +190,7 @@ Important removals:
 * Remove the :program:`2to3` tool and :mod:`!lib2to3` module
   (deprecated in Python 3.11).
 * Remove the :mod:`!tkinter.tix` module (deprecated in Python 3.6).
-* Remove :func:`!locale.resetlocale()`.
```
Comment (hugovk) [inline Doc/whatsnew/3.13.rst:193]: (This would be a good addition to Sphinx Lint.)

File: Doc/whatsnew/3.13.rst [hunk 2]
```diff
@@ -650,583 +654,669 @@ ast
...
+  In order to handle some corner cases, :meth:`~asyncio.Task.uncancel` may now
+  reset the undocumented ``_must_cancel`` flag
+  when the cancellation count reaches zero.
 
   (Inspired by an issue reported by Arthur Tacca in :gh:`116720`.)
 
-* Add :meth:`asyncio.Queue.shutdown` (along with
-  :exc:`asyncio.QueueShutDown`) for queue termination.
-  (Contributed by Laurie Opperman and Yves Duprat in :gh:`104228`.)
+* When :func:`.TaskGroup.create_task` is called on an inactive
```
Comment (hugovk) [inline Doc/whatsnew/3.13.rst]: One more warning: 

> /home/runner/work/cpython/cpython/Doc/whatsnew/3.13.rst:731: WARNING: py:func reference target not found: TaskGroup.create_task [ref.func]

https://github.com/python/cpython/actions/runs/10445076686/job/28920514120?pr=123132

File: Doc/whatsnew/3.13.rst [hunk 3]
```diff
@@ -650,583 +654,669 @@ ast
...
+
+:mod:`dis`
+----------
 
 * Change the output of :mod:`dis` module functions to show logical
   labels for jump targets and exception handlers, rather than offsets.
-  The offsets can be added with the new ``-O`` command line option or
-  the ``show_offsets`` parameter.
+  The offsets can be added with the new
+  :option:`-O <dis --show-offsets>` command line option
```
Comment (hugovk) [inline Doc/whatsnew/3.13.rst]:
```suggestion
  :option:`-O <dis --show-offsets>` command-line option
```

File: Doc/whatsnew/3.13.rst [hunk 4]
```diff
@@ -650,583 +654,669 @@ ast
...
-  the standard format specification mini-language rules for fill, alignment,
-  sign handling, minimum width and grouping. (Contributed by Mark Dickinson
-  in :gh:`111320`.)
+The cyclic garbage collector is now incremental,
+which changes the meaning of the results of
+:meth:`~gc.get_threshold` and :meth:`~gc.set_threshold`
+as well as :meth:`~gc.get_count` and :meth:`~gc.get_stats`.
+
+* For backwards compatibility, :meth:`~gc.get_threshold` continues to return
+  a three-item tuple .
```
Comment (hugovk) [inline Doc/whatsnew/3.13.rst]:
```suggestion
  a three-item tuple.
```

File: Doc/whatsnew/3.13.rst [hunk 5]
```diff
@@ -650,583 +654,669 @@ ast
...
-* Select the default number of worker threads and processes using
+* The default number of worker threads and processes is now selected using
   :func:`os.process_cpu_count` instead of :func:`os.cpu_count`.
   (Contributed by Victor Stinner in :gh:`109649`.)
 
 
-opcode
-------
+:mod:`!opcode`
+--------------
```
Comment (hugovk) [inline Doc/whatsnew/3.13.rst]: These two look like removals, shall we shift them to that section?

File: Doc/whatsnew/3.13.rst [hunk 6]
```diff
@@ -650,583 +654,669 @@ ast
...
-  system. This function is only available on Windows.
+* Add :func:`~os.path.isreserved` to check if a path is reserved
+  on the current system.
+  This function is only available on Windows.
   (Contributed by Barney Gale in :gh:`88569`.)
 
-* On Windows, :func:`os.path.isabs` no longer considers paths starting with
-  exactly one (back)slash to be absolute.
+* On Windows, :func:`~os.path.isabs` no longer considers paths
+  starting with exactly one (back)slash (``\`` or ``/``) to be absolute.
```
Comment (hugovk) [inline Doc/whatsnew/3.13.rst]: Shall we just say "slash"? We show the two types to make it clear:
```suggestion
  starting with exactly one slash (``\`` or ``/``) to be absolute.
```

(Or more explicit:)

```suggestion
  starting with exactly one forward or backslash (``\`` or ``/``) to be absolute.
```

File: Doc/whatsnew/3.13.rst [hunk 7]
```diff
@@ -650,583 +654,669 @@ ast
...
+* :mod:`zipapp` is now supported as a debugging target.
   (Contributed by Tian Gao in :gh:`118501`.)
 
-* ``breakpoint()`` and ``pdb.set_trace()`` now enter the debugger immediately
-  rather than on the next line of code to be executed. This change prevents the
-  debugger from breaking outside of the context when ``breakpoint()`` is positioned
-  at the end of the context.
-  (Contributed by Tian Gao in :gh:`118579`.)
+* Add ability to move between chained exceptions during
+  post mortem debugging in :func:`~pdb.pm` using
```
Comment (hugovk) [inline Doc/whatsnew/3.13.rst]:
```suggestion
  post-mortem debugging in :func:`~pdb.pm` using
```

File: Doc/whatsnew/3.13.rst [hunk 8]
```diff
@@ -650,583 +654,669 @@ ast
...
+  the new ``exceptions [exc_number]`` command for Pdb.
```
Comment (hugovk) [inline Doc/whatsnew/3.13.rst]: Is this "Pdb" the class `Pdb` or module `pdb` or something else?

File: Doc/whatsnew/3.13.rst [hunk 9]
```diff
@@ -616,31 +616,35 @@ New Modules
...
 
-* The constructors of node types in the :mod:`ast` module are now stricter
-  in the arguments they accept, and have more intuitive behaviour when
-  arguments are omitted.
+:mod:`ast`
+----------
+
+* The constructors of node types in the :mod:`ast` module are now
+  stricter in the arguments they accept,
+  with more intuitive behaviour when arguments are omitted.
```
Comment (hugovk) [inline Doc/whatsnew/3.13.rst:647]: In the following paragraph, there's:

> a `ast.expr_context`

Let's make it:

> an `ast.expr_context`

Or:

> an `expr_context`

File: Doc/whatsnew/3.13.rst (hunk 8 above)
Comment (AA-Turner) [reply Doc/whatsnew/3.13.rst]: I think it means the Pdb command-line, I've added a cross-reference which might help things, or we could add clarificatory words.

**Comment by AA-Turner** [comment]
Heading links all gone.

A

**Review by hugovk — APPROVED** [review]
Thank you!

File: Doc/whatsnew/3.13.rst (hunk 1 above)
Comment (hugovk) [reply Doc/whatsnew/3.13.rst:193]: (Requested: https://github.com/sphinx-contrib/sphinx-lint/issues/114)

**Pull request merged**